src/components/Firestore/DocumentPreview/index.tsx

```diff
@@ -101,6 +105,15 @@ const DocumentPreview: React.FC<Props> = ({
               </i>
             </Typography>
           )}
+          {docEmpty && (
+            <Typography
```

Comment:
Can we pull this out into a fn that :97 and this can both use?

Reply:
Done!